src/main/java/com/gluonhq/substrate/target/AbstractTargetConfiguration.java

```diff
@@ -145,11 +145,26 @@ private String getJniPlatform( String os ) {
             default: throw new IllegalArgumentException("No support yet for " + os);
         }
     }
+    /*
+     * Make sure the clibaries needed for linking are available for this particular configuration.
+     * The clibraries path is available by default in GraalVM, but the directory for cross-platform libs may
+     * not exist. In that case, retrieve the libs from our download site.
+     */
+    void ensureClibs (ProjectConfiguration projectConfiguration) throws IOException {
```

Comment:
`private`?

Reply:
my initial goal was to let the specific configurations handle it, but so far this is not needed so it can be private indeed.